CTFd/views.py

```diff
@@ -139,11 +139,29 @@ def team(teamid):
     if get_config('view_scoreboard_if_authed') and not authed():
         return redirect(url_for('auth.login', next=request.path))
     errors = []
+    freeze = get_config('freeze')
     user = Teams.query.filter_by(id=teamid).first_or_404()
     solves = Solves.query.filter_by(teamid=teamid)
-    awards = Awards.query.filter_by(teamid=teamid).all()
-    score = user.score()
-    place = user.place()
+    awards = Awards.query.filter_by(teamid=teamid)
+
+    place = None
+    score = None
+
+    if freeze:
```

Comment:
I think it's a better idea to have user.place() always understand where the user's place is regardless of settings. It makes it a little easier to have config related changes propagate.